addons/account/models/account_move.py

```diff
@@ -1210,16 +1210,18 @@ def _compute_tax_totals(self):
                         ))
                 move.tax_totals = self.env['account.tax']._prepare_tax_totals(**kwargs)
                 if move.invoice_cash_rounding_id:
-                    rounding_amount = move.invoice_cash_rounding_id.compute_difference(move.currency_id, move.tax_totals['amount_total'])
+                    rounding_lines = move.line_ids.filtered(lambda line: line.display_type == 'rounding')
```

Comment:
The dynamic rounding lines are only computed during `create` or `write` and not during `onchange`, this is why we used `move.tax_totals['amount_total']`
Otherwise the value display in the form view won't be correct before saving.

Reply:
yes, right but it's give problem when `write` is called on posted entry.
for example "To Check" option.
and if there is a difference in the cash rounding method like before it was 0.01 and now it's 1.00
then we try to modify entries and it raises error so at the end I can't tick To check.

Reply:
@william-andre

Reply:
This breaks the tax totals with regards to rounding lines when editing the invoice, which is way more used than changing the configuration of a used rounding method.

Reply:
@william-andre Okay, then we do not touch this one, just stop recomputing if it's posted.